deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -340,7 +339,12 @@ spec:
           --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
           --kubefed-namespace=<namespace> \        # namespace where federation is deployed
           -v 2
-    
+     
+     Note: if you're using an openshift cluster for deploying the kubefed-operator then
```

Comment:
What's behind this addition?

Reply:
The issue(https://github.com/openshift/federation-v2-operator/issues/58) filed under the `federation-v2` operator draws my attention to put this thing in README.